docs/tutorials/infer_phys_pars.rst

```diff
@@ -0,0 +1,503 @@
+solutions to the right-hand-side equation for :math:`\Delta\Omega_\mathrm{E}`,
+offset by :math:`180^\circ`. These correspond to rotating the screen by
+:math:`180^\circ` on the sky and this ambiguity in screen orientation cannot be
+resolved using single-telescope data. The angle :math:`\xi`, however, is
+restricted to the range :math:`0^\circ \leq \xi < 180^\circ` (because we use
+the convention that :math:`\xi` refers to the position angle of the *eastern*
+half of the line of lensed images). So, for the purpose of inferring
+:math:`\xi`, it is only necessary to consider one of the two
+:math:`\Delta\Omega_\mathrm{E}` solutions. We use Astropy's
+:py:class:`~astropy.coordinates.Angle` class and its
+:py:meth:`~astropy.coordinates.Angle.wrap_at` method to restrict the value of
+:math:`\xi` to its allowed range.
+
+.. jupyter-execute::
+
+    delta_omega_e = np.arctan(np.tan(chi_e) / np.cos(i_e))
+    xi = delta_omega_e + omega_e
+    xi = Angle(xi).wrap_at(180.*u.deg).deg * u.deg
+
+    print(f'xi: {xi.to(u.deg):8.2f}')
+
+
+The orientation of the pulsar's orbit
+-------------------------------------
+
+Knowing :math:`\xi`, it is possible to retrieve a relation between
+:math:`\Omega_\mathrm{p}` and :math:`i_\mathrm{p}` from the equation
+
+.. math::
+
+    \Omega_\mathrm{p} = \xi - \Delta\Omega_\mathrm{p},
+    \qquad \mathrm{with} \qquad
+    \tan( \Delta\Omega_\mathrm{p} ) = \frac{ \tan( \chi_\mathrm{p} ) }
+                                           { \cos( i_\mathrm{p} ) }.
+
+Again, for a given value of :math:`\chi_\mathrm{p}`, there are two possible
+solutions for :math:`\Delta\Omega_\mathrm{p}`, offset by :math:`180^\circ`.
+Hence, there are two possible :math:`i_\mathrm{p}`--:math:`\Omega_\mathrm{p}`
+relations, offset by :math:`180^\circ` in :math:`\Omega_\mathrm{p}`. We use
+Astropy's :py:class:`~astropy.coordinates.Angle` and
+:py:meth:`~astropy.coordinates.Angle.wrap_at` to restrict the values of
+:math:`\Omega_\mathrm{p}` to its allowed range of
+:math:`0^\circ \leq \Omega_\mathrm{p} < 360^\circ`.
+
+.. jupyter-execute::
+
+    i_p = np.linspace(0., 180., 181) << u.deg
+
+    delta_omega_p1 = np.arctan(np.tan(chi_p) / np.cos(i_p))
+    delta_omega_p2 = delta_omega_p1 + 180.*u.deg
+
+    omega_p1 = xi - delta_omega_p1
+    omega_p2 = xi - delta_omega_p2
+
+    omega_p1 = Angle(omega_p1).wrap_at(360.*u.deg).deg * u.deg
+    omega_p2 = Angle(omega_p2).wrap_at(360.*u.deg).deg * u.deg
+
+The two :math:`i_\mathrm{p}`--:math:`\Omega_\mathrm{p}` relations are
+disjointed at :math:`i_\mathrm{p} = 90^\circ`, where
+:math:`\cos( i_\mathrm{p} )` changes sign. For plotting, we stitch the four
+halves of the two solutions together appropriately to create two continuous
+curves in :math:`i_\mathrm{p}`--:math:`\Omega_\mathrm{p}` space.
+
+.. jupyter-execute::
+
+    ii_ccw = (i_p <= 90.*u.deg)
+    ii_cw =  (i_p >  90.*u.deg)
+
+    omega_p_stitch1 = np.concatenate((omega_p1[ii_ccw], omega_p2[ii_cw]))
```

Comment:
The stitch would then be (I think)
```
omega_p_stich = np.concatenate((omega_p[:, ii_ccw], omega_p[::-1, ii_cw]))
```

Reply:
Done. And I left out the ``_stitch`` part of the variable name.